docs/docs/build/dashboards/dashboards-101.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Dashboards in Rill
+description: Understanding Explore vs Canvas dashboards and when to use each
+sidebar_label: Dashboard Types
+sidebar_position: 00
+---
+
+Rill offers two distinct dashboard experiences, each optimized for different use cases and user needs. The key difference lies in how they handle **metrics views** - the data sources that power your dashboards.
```

Comment:
I would argue that the key difference is that:
- Explore dashboards are _opinionated_ – the data components, visuals, and layout are always the same
- Canvas dashboards are _custom_ – the user composes a set of visualizations however they'd like